### DISCUSSION TITLE
My Github Connection keeps getting lost

### DISCUSSION THREAD

**Discussion by astanciu** [vercel/community#6259] [Account Issue] [answered]
### Question

![image](https://github.com/vercel/community/assets/1675025/8f7993d1-b668-4809-a441-1d03b9a8c138)

I keep losing the connection to my GitHub account. I add it in this screen successfully, it works fine for n number of days, then at some point I see this error on my PRs or commits:
![image](https://github.com/vercel/community/assets/1675025/f110983a-af3e-41e4-99ca-5f1ae1ff6493)

And sure enough, when I check in Vercel, the connection to my account is gone.

What can cause this connection to be removed?

**Comment by astanciu** [comment]
Actually I just figured it out. It looks like I cannot connect one github account to two Vercel accounts (ex: work and personal). Is this a bug?

**Reply by amyegan** [reply, marked as answer]
Hi @astanciu. No, this is not a bug. We do not currently support connecting a GitHub account to more than one Vercel account.
